### DISCUSSION TITLE
6.4.2 프론트엔드 애플리케이션 배포

### DISCUSSION THREAD

**Discussion by rjsdn0006** [fsoftwareengineer/todo-application#120] [질문과 답변]
![1번에러](https://user-images.githubusercontent.com/80525718/171974725-900eae49-2905-4a78-9bf8-28587be420dd.png)

p.381 ~ p.382 진행중 에러가 발생했습니다.
위와 같은 에러가 떠 이전에 다른 분의 글에 답변해 주신 바와 같이 띄어쓰기를 없애고 붙여서 써보았더니 아래와 같이 
환경이름을 짓는것 까진 갔으나, 이후 다른 에러가 발생했습니다.

![2번에러](https://user-images.githubusercontent.com/80525718/171974766-eb7684ae-2880-4191-955f-f4c99497234e.png)

**Comment by fsoftwareengineer** [comment]
t2.micro28s -> t2.micro 로 수정 해 주시기 바랍니다.
